ldms/python/ldmsd/ldmsd_controller

```diff
@@ -61,9 +61,9 @@ import traceback
 import json
 import re
 
-from ldmsd import ldmsd_config, ldmsd_util, ldmsd_request
-from ldmsd.ldmsd_request import LDMSD_Request, LDMSD_Req_Attr
-from ldmsd.ldmsd_request import LDMSD_Req_Attr
+from ldmsd import ldmsd_config, ldmsd_util, ldmsd_communicator
```

Comment:
I think you don't have to import ldmsd_config because you use the connect facility from the ldmsd_communicator module.